pom.xml

```diff
@@ -241,171 +246,243 @@
                 <version>${maven.wagon.plugin.version}</version>
                 <executions>
                     <execution>
-                        <id>download-controller-api</id>
+                        <id>download-protocol-api</id>
                         <phase>generate-sources</phase>
                         <goals>
                             <goal>download-single</goal>
                         </goals>
                         <configuration>
                             <url>https://raw.githubusercontent.com/atomix/api/master</url>
```

Comment:
I think I wrote this before there were any atomix-api releases. Rather than pulling from `master` we should pull from a tag path (e.g. `https://raw.githubusercontent.com/atomix/atomix-api/v0.4.9`). Also, `atomix/api` redirects to `atomix/atomix-api`, but we should use the correct one (`atomix/atomix-api`).